src/Store/DoctrineStore.php

```diff
@@ -55,7 +55,7 @@ private static function normalizeRecordedOn(string $recordedOnAsString, Abstract
         return $recordedOn;
     }
 
-    private static function normalizePlayhead(string $playheadAsString, AbstractPlatform $platform): int
+    private static function normalizePlayhead(string|int $playheadAsString, AbstractPlatform $platform): int
```

Comment:
If we open up the type here we should also do it here: https://github.com/patchlevel/event-sourcing/pull/135/files#diff-17ca8e151b14a68bb5bf016108d5793f3d6bb69ca630897acaf44f2f8c4993c5R33 since it can be `int` as it seems. 

This would mean at this points we should also adjust it:
* https://github.com/patchlevel/event-sourcing/blob/ba3818fef58f7959255281d90e74aabd69d598de/src/Store/SingleTableStore.php#L53
* https://github.com/patchlevel/event-sourcing/blob/ba3818fef58f7959255281d90e74aabd69d598de/src/Store/MultiTableStore.php#L57

Reply:
@DanielBadura I changed it with: `playhead: string|int`